bin/disco_vpc_ui.py

```diff
@@ -84,17 +102,31 @@ def destroy_vpc_command(args):
         sys.exit(2)
 
 
+def update_vpc_command(args):
+    """ handle vpc update command actions"""
+    if args.vpc_name:
+        vpc = DiscoVPC.fetch_environment(environment_name=args.vpc_name)
+    else:
+        vpc = DiscoVPC.fetch_environment(vpc_id=args.vpc_id)
+
+    if vpc:
+        vpc.update(args.dry_run)
+    else:
+        logging.error("No matching VPC found")
+        sys.exit(2)
+
+
 def list_vpc_command(args):
     """ handle list vpcs command actions """
     for vpc_env in DiscoVPC.list_vpcs():
-        line = u"{0}\t{1:<15}".format(vpc_env.id, vpc_env.tags.get("Name", "-"))
+        line = u"{0}\t{1:<15}".format(vpc_env['id'], vpc_env['tags'].get("Name", "-"))
```

Comment:
We use `-` as the default value for a lot of [things](https://github.com/amplifylitco/asiaq/blob/master/bin/disco_aws.py#L266) if we can't look up the proper value. Because there isn't actually a name associated with this VPC, I'd argue for continuing to use `-` for consistency's sake with our other tools. Also, if we can't actually assign the name `default` to the default VPC, I think pretending it has a name when it doesn't would be more confusing.